pkg/templates/images/images.go

```diff
@@ -213,11 +213,11 @@ func FindResource(name string) (Resource, error) {
 
 func baseResources() map[Resource]map[string]string {
 	return map[Resource]map[string]string{
-		CalicoCNI:        {"*": "quay.io/calico/cni:v3.23.5"},
-		CalicoController: {"*": "quay.io/calico/kube-controllers:v3.23.5"},
-		CalicoNode:       {"*": "quay.io/calico/node:v3.23.5"},
+		CalicoCNI:        {"*": "quay.io/calico/cni:v3.26.0"},
+		CalicoController: {"*": "quay.io/calico/kube-controllers:v3.26.0"},
+		CalicoNode:       {"*": "quay.io/calico/node:v3.26.0"},
 		DNSNodeCache:     {"*": "registry.k8s.io/dns/k8s-dns-node-cache:1.22.15"},
-		Flannel:          {"*": "quay.io/coreos/flannel:v0.15.1"},
+		Flannel:          {"*": "docker.io/flannel/flannel:v0.21.3"},
```

Comment:
That's bad, we should mention this in the release note for heads up.

Reply:
They are Sponsored OSS on docker.